### PR TITLE
Feat/247 create dao page ru

### LINKED ISSUE #247: [Localization] Add localization for "Create DAO" page
- [ ] Implement localization for validation errors for DAO Name field
Screenshot 2021-12-22 at 20 51 04
Screenshot 2021-12-22 at 20 50 56

- [ ] "Enter correct URL! validation error
Screenshot 2021-12-22 at 20 51 21

- [ ] Implement translation for DAO section rules
Screenshot 2021-12-22 at 20 51 46

Here are already created keys / values pairs in '/locales/../common.json'
Screenshot 2021-12-22 at 20 53 39
Screenshot 2021-12-22 at 20 52 44

@VERZUOL1 @karpovdmitryod

### PR DESCRIPTION
**Pull request by Izek**

### CODE REVIEW

File: astro_2.0/components/AppHeader/components/NotificationsBell/NotificationsBell.tsx [hunk 1]
```diff
@@ -43,10 +45,30 @@ export const NotificationsBell: VFC<NotificationsBellProps> = ({
   }
 
   return accountId && FEATURE_FLAGS.NOTIFICATIONS ? (
-    <Link href="/notifications" passHref>
-      <div className={cn(styles.root, className)} ref={rootRef}>
-        {renderBellIcon()}
-      </div>
-    </Link>
+    <GenericDropdown
```
Comment (VERZUOL1) [inline astro_2.0/components/AppHeader/components/NotificationsBell/NotificationsBell.tsx:48]: The dropdown was removed from this place by design, so we need a link only

File: astro_2.0/components/AppHeader/components/NotificationsBell/NotificationsBell.tsx [hunk 2]
```diff
@@ -21,6 +22,7 @@ export const NotificationsBell: VFC<NotificationsBellProps> = ({
   className,
 }) => {
   const rootRef = useRef(null);
+  const [open, setOpen] = useState(false);
```
Comment (VERZUOL1) [inline astro_2.0/components/AppHeader/components/NotificationsBell/NotificationsBell.tsx:25]: We don't need to track open state here as we removed dropdown from this component

File: astro_2.0/components/NotificationCard/NotificationCard.module.scss [hunk 3]
```diff
@@ -188,21 +180,9 @@
       display: block;
       flex-shrink: 0;
       margin-right: 14px;
-      //width: 75px;
+      width: 75px;
     }
   }
-
```
Comment (VERZUOL1) [inline astro_2.0/components/NotificationCard/NotificationCard.module.scss:194]: It looks like you reverted an old code here

File: astro_2.0/components/NotificationCard/NotificationCard.tsx [hunk 4]
```diff
@@ -26,56 +27,65 @@ export const NotificationCard: React.FC<NotificationCardProps> = ({
...
+  isDeleteAvailable,
+  isMarkReadAvailable,
+  isMuteAvailable,
   signerId,
-  id,
+  // id,
   status,
 }) => {
   const router = useRouter();
-  const { handleUpdate } = useNotifications();
```
Comment (VERZUOL1) [inline astro_2.0/components/NotificationCard/NotificationCard.tsx:41]: It looks like you reverted old code here

**Review by VERZUOL1 — CHANGES_REQUESTED** [review]

File: astro_2.0/components/AppHeader/components/NotificationsBell/NotificationsBell.tsx (hunk 1 above)
Comment (Izek) [reply astro_2.0/components/AppHeader/components/NotificationsBell/NotificationsBell.tsx:48]: These changes are not from my PR. PR closed, new PR #252

**Pull request closed without merging**